powerline/bindings/lemonbar/powerline-lemonbar.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import time
+import re
+import subprocess
+
+from threading import Lock, Timer
+from argparse import ArgumentParser, REMAINDER
+
+from powerline.lemonbar import LemonbarPowerline
+from powerline.lib.shell import run_cmd
+
+
+if __name__ == '__main__':
+	parser = ArgumentParser(
+		description='Powerline BAR bindings.',
+		usage='%(prog)s [-h] [--i3] [--height HEIGHT] [--interval INTERVAL] [--bar-command CMD] -- args')
```

Comment:
yes, but all `lemonbar` options start with it:

    NAME
           bar - bar ain't recursive

    SYNOPSIS
           bar [-h | -g widthxheight+x+y | -b | -d | -f font | -p | -u pixel | -B color | -F color]